easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb

```diff
@@ -0,0 +1,28 @@
+name = 'Armadillo'
+version = "9.860.1"
+
+homepage = 'https://arma.sourceforge.net/'
+description = """Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
+ a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
+ as well as a subset of trigonometric and statistics functions."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://sourceforge.net/projects/arma/files']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['1603888ab73b7f0588df1a37a464436eb0ff6b1372a9962ee1424b4329f165a9']
+
+dependencies = [
+    ('Boost', '1.71.0'),
+    ('arpack-ng', '3.7.0'),
+]
+
+builddependencies = [('CMake', '3.15.3')]
+
+sanity_check_paths = {
+    'files': ["lib/libarmadillo.%s" % SHLIB_EXT,
+              'share/Armadillo/CMake/ArmadilloConfig.cmake'],
+    'dirs': []
+}
+
```

Comment:
The `sanity_check_paths` do not seem necessary. The easyblock of Armadillo will already take care of them.
```suggestion
```

Reply:
done in c2fdecd